content/en/agent/basic_agent_usage/windows.md

```diff
@@ -319,7 +331,7 @@ While Windows does not offer this exact metric, there is an equivalent option th
 
 ### Monitoring Windows Processes
 
-You can monitor Windows processes via the [Process integration][7]. To set this up on Windows, select the **Process** integration from the list of integrations in the Datadog Agent Manager and edit the configuration.
+You can monitor Windows processes via the [Process integration][8]. To set this up on Windows, select the **Process** integration from the list of integrations in the Datadog Agent Manager and edit the configuration.
```

Comment:
wouldn't we prefer the process agent to the processes integration?

Reply:
valid point, let's address this later though